content/en/integrations/kubernetes_state_core.md

```diff
@@ -58,11 +58,62 @@ datadog:
 ...
 ```
 
+{{% /tab %}}
+{{% tab "Operator" %}}
+
+From the Datadog-Operator v0.7.0
```

Comment:
Maybe we should have this as a note, e.g. `Note: Datadog Operator v.0.7.0 or greater is required.`

Reply:
I move it at the bottom of this tab since it is a note